src/selectors/per-thread/thread.js

```diff
@@ -270,6 +270,19 @@ export function getBasicThreadSelectorsPerThread(
     }
   );
 
+  const getFilteredSampleIndexOffset: Selector<number> = createSelector(
```

Comment:
nit: can you please add a small comment outlining the difference with the previous function?

Reply:
Done.